### PR TITLE
[WIP] Added members drop-in script and dynamic content loader

### PR DESCRIPTION
**Pull request by rishabhgrg**
WIP

### CODE REVIEW

File: post.hbs [hunk 1]
```diff
@@ -33,9 +33,29 @@ into the {body} of the default.hbs template --}}
...
-                </div>
+                {{#if @labs.members}}
+                    {{#if html}}
+                        <div class="post-content">
+                            {{content}}
+                        </div>
+                    {{else}}
+                        <div class="post-content"
+                            data-members='true'
+                            data-members-drop-in-resource="post"
```
Comment (allouis) [inline post.hbs]: Nothing else uses "drop-in" so maybe `data-members-resource-type` might fit better?

File: post.hbs [hunk 2]
```diff
@@ -131,6 +151,11 @@ into the {body} of the default.hbs template --}}
 
 {{!-- The #contentFor helper here will send everything inside it up to the matching #block helper found in default.hbs --}}
 {{#contentFor "scripts"}}
+
+{{#if @labs.members}}
+    <script src="{{asset "js/members-theme-dropin.min.js"}}"></script>
```
Comment (allouis) [inline post.hbs]: Should this be `built/members-theme-dropin.min.js`?

Comment (allouis) [reply post.hbs]: I also think this should be in `default.hbs`

File: post.hbs (hunk 1 above)
Comment (rishabhgrg) [reply post.hbs]: Yeah thats much better 👍 `drop-in` doesn't sound generic too

Comment (allouis) [reply post.hbs]: Cool! @rishabhgrg I'm making this change, so can you update the SDK to work with it?

File: post.hbs (hunk 2 above)
Comment (rishabhgrg) [reply post.hbs]: Yeah it can go there, it already happens implicitly right now as everything in this block is moved to `defaults.hbs`.

Comment (allouis) [reply post.hbs]: Yeah, but it'll only happen on a post right, we want the log in/ out buttons on every page afaik :)

Comment (rishabhgrg) [reply post.hbs]: Ah yes, lets move it 👍

**Pull request merged**